samples/msal-node-samples/standalone-samples/b2c-silent-flow/index.js

```diff
@@ -224,6 +246,9 @@ app.get("/redirect", (req, res) => {
 
         pca.acquireTokenByCode(tokenRequest)
             .then((response) => {
+                console.log(response);
```

Comment:
should these console.log statements be here?

Reply:
Thanks for catching these! Removed.